backend/dal/Services/Admin/Concrete/UserService.cs

```diff
@@ -167,12 +226,10 @@ public Paged<AccessRequest> GetAccessRequestsNoTracking(int page = 1, int quanti
                 .Include(p => p.Roles)
                 .ThenInclude(p => p.Role)
                 .Include(p => p.User)
-                .AsNoTracking();
+                .AsNoTracking()
+                .Where(ar => ar.User != null); // Access Requests with no user cannot be granted.
```

Comment:
We shouldn't be saving access requests without users.  Makes me wonder if the additional filter here is needed.